frontend/test/__support__/server-mocks/collection.ts

```diff
@@ -93,7 +93,7 @@ export function setupCollectionItemsEndpoint({
   collectionItems = [],
   models: modelsParam,
 }: {
-  collection: Collection;
+  collection: Pick<Collection, "id">;
```

Comment:
If you look at the implemetnation we only need `id`, so in tests, it's more convenient to not have to construct the whole collection just to pass it here.